Felienne/hedy
# Implements the support for multiple admin users

**Description**
In this PR we make a small change to how we handle the admin user by implementing support for multiple admins. The `is_admin` function is changed to look into the environment for the `ADMIN_USERS` list and then verify if the current user is in there. If so, the user is an admin. This introduces some complications for our contributors and the following changes to then environment variable should be made to get this to work:
- Change `ADMIN_USER` to `ADMIN_USERS`
- Change the value from `"username"` to `["username"]`

**Fixes**
This PR fixes #3160.

**How to test**
Make sure you make the changes as described above and then verify that all admin functionality is still working. **Important:** we do have to make the same changes in Heroku before merging.
